src/main/java/seedu/duke/command/DeleteNoteCommand.java

```diff
@@ -46,22 +48,22 @@ public DeleteNoteCommand(String title) {
 
     @Override
     public String execute() {
-        ArrayList<Note> deletedListTitle;
         try {
             // If there is no title, delete note by index. Else delete by title.
             if (title.isBlank()) {
                 String deletedTitle = notebook.getNotes().get(index).getTitle();
                 notebook.deleteNote(index);
-                return COMMAND_SUCCESSFUL_MESSAGE + deletedTitle;
+
+                return Formatter.formatString(COMMAND_SUCCESSFUL_MESSAGE + deletedTitle, false);
             } else {
-                deletedListTitle = (ArrayList<Note>) notebook.getNotes().stream()
-                        .filter((s) -> s.getTitle().toLowerCase().contains(title.toLowerCase()))
-                        .collect(toList());
-                notebook.deleteNote(deletedListTitle.get(0).getTitle());
-                return COMMAND_SUCCESSFUL_MESSAGE + deletedListTitle.get(0).getTitle();
+                notebook.deleteNote(title);
+
+                return Formatter.formatString(COMMAND_SUCCESSFUL_MESSAGE + title, false);
             }
-        } catch (IndexOutOfBoundsException | NullPointerException | ClassCastException exception) {
-            return COMMAND_UNSUCCESSFUL_MESSAGE;
+        } catch (SystemException exception) {
+            return Formatter.formatString(exception.getMessage(), false);
```

Comment:
Is systemException thrown anywhere in Formatter?

Reply:
as of now no